compiler/test/formatter_outputs/list_sugar.gr

```diff
@@ -0,0 +1,36 @@
+let list = [1]
+
+let list2 = [1,2]
+
+let list3 = [1,2,3]
+
+let list4 = [[1,2],[3,4]]
+
+let list5 = [[1, 2, 3 ], [3, 4, 5 ], [5, 6, 7 ] ]
```

Comment:
@marcusroberts What's going on with the spaces here? (from the filename, this is what the formatter produces, right?)

Reply:
That is a good question, taking a look now

Reply:
Well spotted, it was an out of date test not in the suite!